src/app/demo/elements/element-color/element-color.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+
+@Component({
+  selector: 'app-element-color',
+  standalone: true,
+  imports: [CommonModule, RouterModule],
```

Comment:
RouterModule needed?

Reply:
those component in use router link  so this component need router Module